travis_configure.sh

```diff
@@ -38,12 +38,10 @@ before_install()
             build
 
         # jfrog dependency
-        if [[ "$MAKE_TARGET" =~ "deb" ]]; then
-            if [[ -n "$TRAVIS_TAG" && $TRAVIS_REPO_SLUG == "sociomantic-tsunami/libdrizzle-redux" ]]; then
```

Comment:
So you don't need the check on `TRAVIS_REPO_SLUG` any more?

Reply:
The check was done to only install `jfrog` for upstream builds. But while testing it is quite convenient to be able to test deployment for your own repo, cf. the description for #266